Makefile

```diff
@@ -26,10 +26,14 @@ BUILD_PROXY += --build-arg https_proxy=$(https_proxy)
 RUN_PROXY += --env https_proxy=$(https_proxy)
 endif
 
+RUN_OS_VERSION= --env OS_VERSION="$(OS_VERSION)"
+RUN_AGENT_VERSION= --env AGENT_VERSION=$(AGENT_VERSION)
```

Comment:
Missing quotes for the variable value (for consistency with the other variables around this one)?